bundles/org.eclipse.passage.lbc.jetty/META-INF/MANIFEST.MF

```diff
@@ -15,4 +15,4 @@ Require-Bundle: org.eclipse.core.runtime;bundle-version="0.0.0",
  org.eclipse.passage.lic.net;bundle-version="0.0.0"
 Bundle-ActivationPolicy: lazy
 Bundle-Activator: org.eclipse.passage.lbc.internal.jetty.FlsJettyActivator
-Export-Package: org.eclipse.passage.lbc.jetty
```

Comment:
do we hide an API here? In this case we should have a major increment

Reply:
From recent resource planning it was defined that LBC components are illustrative and most probably should go as provisional API completely. What do you think?